src/test/java/RozetkaSergey/RozetkaPages/RozetkaPages5_34.java

```diff
@@ -64,7 +65,26 @@ public WebElement affirmDelete() {
     public WebElement closeAdvertisement (){return getElement("//span[@class='exponea-close-cross']");}
     public WebElement listToleft (){return getElement("//button[@aria-label='Предыдущий слайд']");}
     public WebElement listToRight (){return getElement("//button[@aria-label='Следующий слайд']");}
+    public WebElement clickOnThreeStrips (){return getElement("//button[@class='header__button'][contains(@aria-label, 'Открыть меню')]");}
+    public WebElement chooseCity (){return  getElement("//button[@class='city-toggle button button--small button--white']");}
+    public List<WebElement> listOfCites (){return getElements("//a[@class='header-location__popular-link']");}
+    public WebElement klickAsept (){return getElement("//button[@class='button button_size_medium button_color_green']");}
+    public WebElement changLanguageButton(){return getElement("//li[@class='lang__item __item']");}
+    public WebElement fiendButton   () {return getElement("/html/body/app-root/div/div[1]/rz-header/header/div/div/div/form/button");}
+    public WebElement contactButton (){ return getElement("//ul[@class='main-links__list']//a[@href='https://rozetka.com.ua/contacts/']");}
+    public List<WebElement> listOfTelefonNumbers (){return getElements("//section[@class='contacts-section']//div[@class='contacts-main__cell']//a[@class='contacts-main__number']");}
 
+
+
+    public void checkingListOfCities(){
+        List<WebElement> cities = listOfCites();
+        for (int i = 0; i<5; i++){
+            System.out.println(cities.get(i).getText());
+        }
+
```

Comment:
its a shame to have so many empty lines!)

Reply:
done